app/angular/src/client/preview/angular-beta/AbstractRenderer.ts

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable no-undef */
+import { enableProdMode, NgModule, PlatformRef } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { BehaviorSubject, Subject } from 'rxjs';
+import { ICollection, StoryFnAngularReturnType } from '../types';
+import { Parameters } from '../types-6-0';
+import { createStorybookModule, getStorybookModuleMetadata } from './StorybookModule';
+
+type StoryRenderInfo = {
+  storyFnAngular: StoryFnAngularReturnType;
+  moduleMetadataSnapshot: string;
+};
+
+// platform must be init only if render is called at least once
+let platformRef: PlatformRef;
```

Comment:
Should this be `_platformRef` instead so devs won't tend to use this variable instead of `getPlatform`? 
Highly opinionated code suggestion 😄 I don't mind if you leave it as is

To me `_name` always feels like "don't touch this, use something else"

Reply:
Yes, that's the idea `don't touch this, use something else` :p 

I haven't found a clean way to make the new `platformRef` 

but to avoid mistakes with `storyshots` the `platformBrowserDynamic` must be call only if the render is called at least once.

I thought of doing it in the factory to give it to all Renderer instance but it was more code, not much clearer so I hid it here 😈 
But I take all your ideas :)